### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by ccoVeille**
- **Fix typos in code and tests**
- **Fix acronyms and brand names**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -143,14 +143,14 @@ func main() {
...
 		Compare:     *comparePtr,
 
-		// Do not expose publically.
+		// Do not expose publicly.
 		DisableTableOutput: false,
 	}
 	exitCode, err := app.Run(os.Stdout, options)
 	if err != nil {
 		msg := err.Error()
-		if errors.Is(err, parse.ErrNotParseable) {
```
Comment (mfridman) [inline main.go:152]: Given this is a pre-v1 release, I'm comfortable making this breaking change.

**Comment by ccoVeille** [comment]
> I merged the other 2 PR's, if you want to rebase/merge please go ahead, otherwise I can do that 🤝

I will sure

**Comment by ccoVeille** [comment]
It did it for record.

**Comment by mfridman** [comment]
Thanks for a set of nice PR's to tidy up the project @ccoVeille 🎉

**Pull request merged**